timesketch/frontend-ng/src/views/Sketch.vue

```diff
@@ -355,11 +202,163 @@ limitations under the License.
       </v-menu>
     </v-app-bar>
 
-    <!-- Canvas (main) view -->
-    <router-view
-      v-if="sketch.status && hasTimelines && !isArchived"
-      @setTitle="(title) => (this.title = title)"
-    ></router-view>
+    <!-- Left panel -->
+    <v-navigation-drawer
+      v-if="showLeftPanel && hasTimelines && !isArchived"
+      app
+      clipped
+      permanent
+      :width="navigationDrawer.width"
+      hide-overlay
+      ref="drawer"
+    >
+      <!-- Dialog for adding a scenario -->
+      <v-dialog v-model="scenarioDialog" max-width="500px">
+        <v-card>
+          <div class="pa-3">
+            <h3>Investigative Scenarios</h3>
+            <v-select
+              v-model="selectedScenario"
+              :items="scenarioTemplates"
+              item-text="name"
+              return-object
+              label="Select a scenario"
+              outlined
+              class="mt-3"
+            ></v-select>
+            <div v-if="selectedScenario">
+              {{ selectedScenario.description }}
+            </div>
+          </div>
+          <v-divider></v-divider>
+          <v-card-actions>
+            <v-spacer></v-spacer>
+            <v-btn @click="scenarioDialog = false" text> Cancel </v-btn>
+            <v-btn text color="primary" :disabled="!selectedScenario" @click="addScenario(selectedScenario.id)">
+              Add
+            </v-btn>
+          </v-card-actions>
+        </v-card>
+      </v-dialog>
+
+      <v-tabs v-model="leftPanelTab" grow>
+        <v-tab v-for="item in leftPanelTabItems" :key="item"> {{ item }} </v-tab>
+      </v-tabs>
+      <v-divider></v-divider>
+      <v-tabs-items v-model="leftPanelTab">
+        <v-tab-item :transition="false">
+          <router-link
+            :to="{ name: 'Explore', params: { sketchId: sketchId } }"
+            tag="div"
```

Comment:
According to the console error and [vue docs](https://router.vuejs.org/guide/migration/#Removal-of-event-and-tag-props-in-router-link-) the `tag` prop is deprecated and will be removed in Vue Router 4. Use the `v-slot` API instead.

Reply:
Good catch, done.